views/pages/sqlInjection.gohtml

```diff
@@ -10,8 +10,8 @@
     </h4>
     <form method="{{.Method}}" action="{{$base}}{{.URL}}/unsafe" target="_blank">
       <div class="form-group">
-        <label>SELECT "${input}" as "test";</label>
-        <input name="input" class="form-control" value="Robert&quot;); DROP TABLE Students;--" />
+        <label>SELECT '${input}' as 'test';</label>
+        <input name="input" class="form-control" value="Robert&apos;; DROP TABLE Students;--" />
```

Comment:
Are you sure the change to &apos; is correct?

Reply:
@benjivesterby I see in the backend the query arguments are wrapped in single quotes, therefore, I updated them in the template. If the input value has a double quote the argument is not split as expected, respectively for unsafe and safe modes the final query is the same.

Reply:
Gotcha